### PR TITLE
[Enhancement][ESS] Add a note in Manage detection alerts -> Edit drop-down filter controls 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/4938

Preview: [Manage detection alerts | Edit drop-down filter controls](https://security-docs_bk_5003.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#drop-down-filter-controls)

Twin Serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/312

### CODE REVIEW

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -61,7 +61,8 @@ image::images/alert-page-dropdown-controls.png[Alerts page with drop-down contro
 ====
 * You can have a maximum of four controls on the Alerts page.
 * You can't remove the *Status* control.
-* If you make any changes to the controls, you _must_ save the pending changes for them to persist.
+* If you make any changes to the controls, you _must_ save the pending changes for them to persist. 
+* Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.
```
Comment (logeekal) [inline docs/detections/alerts-ui-manage.asciidoc:65]: > If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.

Somehow word `reapplied` seems not correct. How about below?

If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations and see default Controls.

**Review by logeekal — COMMENTED** [review]
Just one small comment.

File: docs/detections/alerts-ui-manage.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/detections/alerts-ui-manage.asciidoc:65]: I think we can just go with "you'll lose your customizations". The part about your controls reverting to their default state is implied.

Comment (logeekal) [reply docs/detections/alerts-ui-manage.asciidoc:65]: Works for me. Thank you.

Comment (nastasha-solomon) [reply docs/detections/alerts-ui-manage.asciidoc:65]:
```suggestion
* Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations.
```

**Review by logeekal — DISMISSED** [review]
Thanks for making these changes. 🚀 LGTM.

**Review by joepeeples — APPROVED** [review]
LGTM!

**Pull request merged**